app/src/main/res/layout/activity_products_page.xml

```diff
@@ -109,12 +113,31 @@
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/categories"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="15dp"
+        android:layout_height="0dp"
+        app:layout_constraintHeight_percent="0.62"
+        android:padding="15dp"
```

Comment:
a lot of hard-coded dimensions in this file, what do you think about moving them to the dimensions file?

Reply:
That was the plan, I just wanted you guys to be able to workk with the new design